pom.xml

```diff
@@ -451,87 +658,133 @@
 
   </profiles>
 
-    <dependencies>
-      <dependency>
-        <groupId>org.apache.accumulo</groupId>
-        <artifactId>accumulo-core</artifactId>
-        <version>${accumulo.version}</version>
-      </dependency>
-      <dependency>
-        <groupId>org.apache.hadoop</groupId>
-        <artifactId>hadoop-common</artifactId>
-        <version>2.7.2</version>
-      </dependency>
-      <dependency>
-        <groupId>junit</groupId>
-        <artifactId>junit</artifactId>
-        <version>4.12</version>
-      </dependency>
-      <dependency>
-        <groupId>org.apache.accumulo</groupId>
-        <artifactId>accumulo-minicluster</artifactId>
-        <version>${accumulo.version}</version>
-        <scope>test</scope>
-      </dependency>
-      <!-- Replaces Accumulo dependency on Apache Commons 3.2, which does not use Java Generics. -->
-      <dependency>
-        <groupId>org.apache.commons</groupId>
-        <artifactId>commons-collections4</artifactId>
-        <version>4.1</version>
-      </dependency>
-
-      <!-- Used by some JSON functions in D4M Matlab. -->
-      <dependency>
-        <groupId>org.json</groupId>
-        <artifactId>org.json</artifactId>
-        <version>chargebee-1.0</version>
-        <scope>runtime</scope>
-        <optional>true</optional>
-      </dependency>
-
-      <!-- Upgrading to 4.2.0-incubating is impossible without upgrading Zipkin inside Accumulo. -->
-      <dependency>
-        <groupId>org.apache.htrace</groupId>
-        <artifactId>htrace-zipkin</artifactId>
-        <version>3.1.0-incubating</version>
-        <scope>runtime</scope>
-        <optional>true</optional>
-      </dependency>
-
-      <!--<dependency>-->
-        <!--<groupId>commons-codec</groupId>-->
-        <!--<artifactId>commons-codec</artifactId>-->
-        <!--<version>1.7</version>-->
-      <!--</dependency>-->
-
-      <!--<dependency>-->
-        <!--<groupId>org.apache.htrace</groupId>-->
-        <!--<artifactId>htrace-core</artifactId>-->
-        <!--<version>3.1.0-incubating</version>-->
-        <!--<scope>runtime</scope>-->
-      <!--</dependency>-->
-
-      <!-- Accumulo server-side dependencies-->
-      <!--<dependency>-->
-        <!--<groupId>org.apache.commons</groupId>-->
-        <!--<artifactId>commons-math3</artifactId>-->
-        <!--<version>3.1.1</version>-->
-        <!--<scope>runtime</scope>-->
-      <!--</dependency>-->
-
-      <dependency>
-        <groupId>com.googlecode.matrix-toolkits-java</groupId>
-        <artifactId>mtj</artifactId>
-        <version>1.0.4</version>
-      </dependency>
-
-      <dependency>
-        <groupId>org.apache.accumulo</groupId>
-        <artifactId>accumulo-tracer</artifactId>
-        <version>${accumulo.version}</version>
-        <scope>runtime</scope>
-      </dependency>
-
-    </dependencies>
+  <dependencies>
```

Comment:
Main differences in this section: added exclusions to accumulo dependency, indentation difference, junit has test scope, using versions from top, accumulo-tracer is commented out (is accumulo-tracer really needed? I couldn't find it referenced anywhere and everything seems to run fine without it).

Reply:
accumulo-tracer was used in an older version of Accumulo. It has been succeeded by HTrace. Let's remove accumulo-tracer.

Reply:
Removed accumulo-tracer and some other dependencies that were commented out, as well as the exclusions to accumulo-core.